package.json

```diff
@@ -7,7 +7,8 @@
     "setup-dbs": "psql -f ./db/setup.sql",
     "seed": "node ./db/seeds/run-seed.js",
     "test": "jest",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "scrap": "psql -f ./scrap.sql"
```

Comment:
This looks like something you are personally using whilst working on this repo - it should not necessarily be included in production code.